optaplanner-examples/src/main/java/org/optaplanner/examples/conferencescheduling/score/ConferenceSchedulingConstraintProvider.java

```diff
@@ -119,9 +119,12 @@ protected Constraint roomConflict(ConstraintFactory factory) {
     }
 
     protected Constraint speakerUnavailableTimeslot(ConstraintFactory factory) {
-        return factory.forEach(Talk.class)
-                .filter(Talk::hasAnyUnavailableSpeaker)
-                .penalizeConfigurable(SPEAKER_UNAVAILABLE_TIMESLOT, Talk::getDurationInMinutes);
+        return factory.forEachIncludingNullVars(Talk.class)
```

Comment:
We want this constraint to also work in CH, when some of the variables are null. Not doing this would effectively have enabled this constraint in LS only.